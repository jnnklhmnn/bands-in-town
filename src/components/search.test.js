import React from 'react';
import Search from './Search.js';
import renderer from 'react-test-renderer';

test('Search should match Snapshot', () => {

    const component = renderer.create(
        <Search />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});