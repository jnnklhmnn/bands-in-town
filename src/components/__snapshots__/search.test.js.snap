// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search should match Snapshot 1`] = `
<div
  className="bitcontent"
>
  <div
    className="inputsection"
  >
    <div>
      <form
        onSubmit={[Function]}
      >
        <input
          className="input"
          onChange={[Function]}
          placeholder="Search for Band Name"
          type="text"
          value=""
        />
        <button
          className="button"
          disabled={true}
          type="submit"
        >
           Go!
        </button>
      </form>
    </div>
    <div>
      <button
        className="button"
        disabled={true}
        onClick={[Function]}
      >
        search last band again
      </button>
    </div>
  </div>
</div>
`;
